<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>441</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_Main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionFile_Open"/>
    <addaction name="actionFile_New"/>
    <addaction name="actionFile_Save"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="actionEdit_Cut"/>
    <addaction name="actionEdit_Copy"/>
    <addaction name="actionEdit_Paste"/>
    <addaction name="actionEdit_Undo"/>
    <addaction name="actionEdit_Redo"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_SeletcAll"/>
    <addaction name="actionEdit_Clear"/>
   </widget>
   <widget class="QMenu" name="menu_Format">
    <property name="title">
     <string>格式(M)</string>
    </property>
    <widget class="QMenu" name="menu_Format_Lang">
     <property name="title">
      <string>界面语言</string>
     </property>
     <addaction name="actionLang_CN"/>
     <addaction name="actionLang_EN"/>
    </widget>
    <addaction name="actionFont_Bold"/>
    <addaction name="actionFont_Italic"/>
    <addaction name="actionFont_Underline"/>
    <addaction name="separator"/>
    <addaction name="actionSys_ToggleText"/>
    <addaction name="menu_Format_Lang"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Format"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFile_New"/>
   <addaction name="actionFile_Open"/>
   <addaction name="actionFile_Save"/>
   <addaction name="separator"/>
   <addaction name="actionEdit_Cut"/>
   <addaction name="actionEdit_Copy"/>
   <addaction name="actionEdit_Paste"/>
   <addaction name="separator"/>
   <addaction name="actionEdit_Undo"/>
   <addaction name="actionEdit_Redo"/>
   <addaction name="separator"/>
   <addaction name="actionFont_Bold"/>
   <addaction name="actionFont_Italic"/>
   <addaction name="actionFont_Underline"/>
   <addaction name="separator"/>
   <addaction name="actionLang_CN"/>
   <addaction name="actionLang_EN"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionEdit_Cut">
   <property name="icon">
    <iconset>
     <normalon>menu-ico/Cut.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切到粘贴板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionEdit_Copy">
   <property name="icon">
    <iconset>
     <normalon>menu-ico/Copy.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制到粘贴板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionEdit_Paste">
   <property name="icon">
    <iconset>
     <normalon>menu-ico/Paste.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>从粘贴板粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFont_Bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
  </action>
  <action name="actionFont_Italic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
  </action>
  <action name="actionFont_Underline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下滑线</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>menu-ico/Close.png</normaloff>menu-ico/Close.png</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭本窗口</string>
   </property>
  </action>
  <action name="actionSys_ToggleText">
   <property name="text">
    <string>显示按钮文字</string>
   </property>
   <property name="toolTip">
    <string>显示工具栏按钮文字</string>
   </property>
  </action>
  <action name="actionEdit_Clear">
   <property name="icon">
    <iconset>
     <normaloff>menu-ico/New.png</normaloff>menu-ico/New.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空</string>
   </property>
  </action>
  <action name="actionEdit_Undo">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销上次编辑操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionEdit_Redo">
   <property name="text">
    <string>重做</string>
   </property>
   <property name="toolTip">
    <string>重做上次操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionEdit_SeletcAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFile_New">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFile_Open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFile_Save">
   <property name="icon">
    <iconset>
     <normaloff>menu-ico/Save.png</normaloff>menu-ico/Save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存修改</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLang_CN">
   <property name="text">
    <string>中文</string>
   </property>
   <property name="toolTip">
    <string>中文界面</string>
   </property>
  </action>
  <action name="actionLang_EN">
   <property name="text">
    <string>English</string>
   </property>
   <property name="toolTip">
    <string>English Interface</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>undoAvailable(bool)</signal>
   <receiver>actionEdit_Undo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>redoAvailable(bool)</signal>
   <receiver>actionEdit_Redo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>modificationChanged(bool)</signal>
   <receiver>actionFile_Save</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit_Undo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit_Redo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit_SeletcAll</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit_Paste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit_Cut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit_Copy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit_Clear</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
